mne/time_frequency/tests/test_psd.py

```diff
@@ -92,21 +90,82 @@ def test_psd_epochs():
     picks = pick_types(epochs.info, meg='grad', eeg=False, eog=True,
                        stim=False, include=include, exclude='bads')
     psds, freqs = compute_epochs_psd(epochs[:1], fmin=2, fmax=300,
-                                     n_fft=n_fft, picks=picks)
+                                     wel_n_fft=n_fft, picks=picks,
+                                     method='welch')
 
     psds_t, freqs_t = compute_epochs_psd(epochs_full[:1], fmin=2, fmax=300,
-                                         tmin=tmin, tmax=tmax,
-                                         n_fft=n_fft, picks=picks)
+                                         tmin=tmin, tmax=tmax, method='welch',
+                                         wel_n_fft=n_fft, picks=picks)
     # this one will fail if you add for example 0.1 to tmin
-    assert_array_almost_equal(psds, psds_t, 27)
+    assert_array_almost_equal(psds, psds_t, 20)
```

Comment:
use relative comparisons. power of signals in the range of 1e-13 will be 1e-26 so you're check might not check anything :)